npm_and_yarn/spec/fixtures/projects/npm6/npmrc_auth_token_with_space/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "{{ name }}",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no\\ test\ specified\" && exit 1",
```

Comment:
note to myself: Why is github formatter showing this in red? Is it a bug in syntax highligher or a funky character not apparent to normal human eyes? Doublecheck the UTF character byte code...

Reply:
It sounds like that should be a double slash and that's why GitHub is showing it in red?